db-connector/src/main/java/uk/nhs/adaptors/connector/configuration/BaseConfiguration.java

```diff
@@ -0,0 +1,42 @@
+package uk.nhs.adaptors.connector.configuration;
+
+import java.util.List;
+
+import javax.sql.DataSource;
+
+import org.jdbi.v3.core.Jdbi;
+import org.jdbi.v3.core.mapper.RowMapper;
+import org.jdbi.v3.core.spi.JdbiPlugin;
+import org.jdbi.v3.sqlobject.SqlObjectPlugin;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.datasource.TransactionAwareDataSourceProxy;
+
+import uk.nhs.adaptors.connector.dao.PatientMigrationRequestDao;
+
+@Configuration
+@EntityScan(basePackages = {"uk.nhs.adaptors.connector"})
+@ComponentScan(basePackages = {"uk.nhs.adaptors.connector"})
+public class BaseConfiguration {
+
+    @Bean
+    public Jdbi jdbi(DataSource ds, List<JdbiPlugin> jdbiPlugins, List<RowMapper<?>> rowMappers) {
+        TransactionAwareDataSourceProxy proxy = new TransactionAwareDataSourceProxy(ds);
+        Jdbi jdbi = Jdbi.create(proxy);
+        jdbiPlugins.forEach(jdbi::installPlugin);
```

Comment:
should we add Postgres JDBI Plugin? 
https://jdbi.org/apidocs/org/jdbi/v3/postgres/PostgresPlugin.html